src/main/kotlin/com/bardsoftware/backend/fleet/rmarkdown/Publisher.kt

```diff
@@ -0,0 +1,72 @@
+package com.bardsoftware.backend.fleet.rmarkdown
+
+import com.google.api.core.ApiFutureCallback
+import com.google.api.core.ApiFutures
+import com.google.api.gax.rpc.ApiException
+import com.google.cloud.ServiceOptions
+import com.google.cloud.pubsub.v1.Publisher
+import com.google.protobuf.ByteString
+import com.google.pubsub.v1.PubsubMessage
+import com.google.pubsub.v1.TopicName
+
+object PublisherExample {
```

Comment:
We will need the publisher but not in the form of main function. Please convert this code to a class which takes topic name in the constructor arguments and provides a method to publish a message.